QueAns/views.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponse
+from .models import Voting,Meetings,Satisfaction
+from datetime import datetime
+
+
+def upvote(request,user_id,question_id):
+    if request.method == "POST":
+        meeting = Meetings.objects.filter(question_id=question_id)
+        if meeting.end_time < datetime.now():
+            if Voting.objects.filter(user_id=user_id,question_id=question_id).count()<1:
+                Voting.objects.create(user_id=user_id,question_id=question_id,vote=1)
```

Comment:
Is this behavior intended?

Reply:
I will try and fix this by changing the attribute vote instead of creating a new entry in both the upvote and the downvote case